src/MessagePack/MessagePackReader.cs

```diff
@@ -779,8 +779,7 @@ private string ReadStringSlow(int byteLength)
             // We need to concatenate all the bytes together into one array.
             byte[] byteArray = ArrayPool<byte>.Shared.Rent(byteLength);
```

Comment:
Is there a reason you don't put the `Rent` \ `Return` inside a try-finally block?

Reply:
No reason. That would make a good change.